src/pages/NewsArticles.jsx

```diff
@@ -0,0 +1,48 @@
+import { useDispatch, useSelector } from "react-redux";
+import ArticleList from "../components/section/ArticleList";
+import ArticlePublisher from "../components/section/ArticlePublisher";
+import ArticlesSearch from "../components/section/ArticlesSearch";
+import Wrapper from "../components/layouts/Wrapper";
+import { updateArticles, updatePublisher, loadingData } from "../features/news";
+import { useEffect } from "react";
+import { getSources, getTopHeadlines } from "../features/api";
+import ArticleListSkeleton from "../components/section/ArticleListSkeleton";
+import Footer from "../components/layouts/Footer";
+import Navbar from "../components/layouts/Navbar";
+
+const NewsArticles = () => {
+  const inputValue = useSelector((state) => state.news.inputValue);
+  const loading = useSelector((state) => state.news.loading);
```

Comment:
Extracting values would help you write a few lines of codes, Check this suggestion: 
``` suggestion
  const {inputValue, loading} = useSelector((state) => state.news);
```